packages/csharp/PortingAssistant/PortingAssistant.Telemetry/Models/MetricsBase.cs

```diff
@@ -0,0 +1,14 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace PortingAssistant.Telemetry.Model
+{
+    public class MetricsBase
+    {
+        [JsonConverter(typeof(StringEnumConverter))]
+        public MetricsType MetricsType { get; set; }
+        public string PortingAssistantExtensionVersion { get; set; }
```

Comment:
change it to porting assistant version

Reply:
Are we passing App version from front end? I didn't see it in the front end code. We probably need it